cli/commands.js

```diff
@@ -22,8 +22,9 @@ if (!shell.test('-f', userPackageJSONPath)) {
 }
 
 const loadConfigAndDo = (action, optionalConfig) => {
+  const flags = program.args.filter(item => typeof item === 'string');
   const config = kytConfigFn(optionalConfig);
-  action(config, program);
+  action(config, program, flags);
```

Comment:
It looks like `program` isn't actually used be any of the actions, and instead just need `flags`. Can you double check that, and if true, maybe we can simplify this signature to just this:

```js
action(config, flags);
```